ext/node/ops/os/mod.rs

```diff
@@ -50,7 +50,10 @@ where
 {
   {
     let permissions = state.borrow_mut::<P>();
-    permissions.check_sys("userInfo", "node:os.userInfo()")?;
+    permissions.check_sys("uid", "node:os.userInfo()")?;
+    permissions.check_sys("gid", "node:os.userInfo()")?;
+    permissions.check_sys("homedir", "node:os.userInfo()")?;
+    permissions.check_sys("username", "node:os.userInfo()")?;
```

Comment:
Can you give me a hand here and explain why we need to check for all of these permissions when getting the user name?

Reply:
It only gets called by `node:os.userInfo()`, which returns all that information, but I hadn't noticed it would already check for the other permissions, sorry.

Reply:
I fixed this, but it may not have been obvious, sorry.